tags/Coveo/CoveoAnalyticsLogEvent/Coveo Analytics Log Event.tpl

```diff
@@ -0,0 +1,453 @@
+        "valueValidators": [
+          {
+            "type": "NON_EMPTY"
+          }
+        ],
+        "displayName": "Content ID Key",
+        "defaultValue": "permanentid",
+        "simpleValueType": true,
+        "name": "contentIdKey",
+        "type": "TEXT"
+      },
+      {
+        "help": "The value of the field used to trace back the current page in the Coveo Index.",
+        "valueValidators": [
+          {
+            "type": "NON_EMPTY"
+          }
+        ],
+        "displayName": "Content ID Value",
+        "simpleValueType": true,
+        "name": "contentIdValue",
+        "type": "TEXT"
+      }
+    ]
+  },
+  {
+    "enablingConditions": [
+      {
+        "paramName": "eventType",
+        "type": "EQUALS",
+        "paramValue": "custom"
+      }
+    ],
+    "displayName": "Custom Event Metadata",
+    "name": "Custom Event Metadata",
+    "groupStyle": "ZIPPY_OPEN",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "help": "",
+        "valueValidators": [
+          {
+            "type": "NON_EMPTY"
+          }
+        ],
+        "displayName": "Event Type",
+        "simpleValueType": true,
+        "name": "customEventType",
+        "type": "TEXT"
+      },
+      {
+        "help": "",
+        "valueValidators": [
+          {
+            "type": "NON_EMPTY"
+          }
+        ],
+        "displayName": "Event Value",
+        "simpleValueType": true,
+        "name": "customEventValue",
+        "type": "TEXT"
+      }
+    ]
+  },
+  {
+    "displayName": "Document Metadata",
+    "name": "Document Metadata",
+    "groupStyle": "ZIPPY_OPEN",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "help": "The current language of the page's content. If not properly detected, will fallback on \"en\".",
+        "displayName": "Language",
+        "simpleValueType": true,
+        "name": "language",
+        "type": "TEXT",
+        "valueValidators": [
+          {
+            "type": "NON_EMPTY"
+          }
+        ]
+      },
+      {
+        "help": "(Optional) The current URL of the page. If not set, \"window.location\" will be used.",
+        "displayName": "Location",
+        "simpleValueType": true,
+        "name": "location",
+        "type": "TEXT"
+      },
+      {
+        "help": "(Optional) The title of the document. If not set, \"document.title\" will be used.",
+        "displayName": "Title",
+        "simpleValueType": true,
+        "name": "title",
+        "type": "TEXT"
+      },
+      {
+        "help": "(Optional) Boolean representing whether the current visit is from an anonymous user.",
+        "displayName": "Is Anonymous",
+        "simpleValueType": true,
+        "name": "isAnonymous",
+        "type": "TEXT"
+      }
+    ]
+  },
+  {
+    "help": "Custom data to send alongside the custom event",
+    "displayName": "Custom Data",
+    "name": "customData",
```

Comment:
Is there a reason you opted for the Param Table rather than the Simple Table? Seems like unnecessary UI complication, since it's just two text input columns.

Reply:
I had not found `Require column values to be unique`, but now I did 😄 I will change it to the Simple one

Reply:
Yeah, it's a bit hidden :D